tests/page-objects/login-page.ts

```diff
@@ -24,6 +27,8 @@ export class LoginPage {
       asUser.password,
     );
     await this.page.locator('input.btn.btn-primary.btn-submit').click();
+
+    await this.page.waitForURL('https://archive.org/');
```

Comment:
Suggestion: We'll very likely need to login to many different URLs if we're going to run E2E tests on review apps and/or staging servers. So this "base URL" should be an easy to see and set variable, not a constant, or hard-coded value.

Ideally, this should be a be an command line option, or maybe environment variable?

Reply:
updated